language-server/update-info.log

```diff
@@ -1,2 +1,2 @@
-Archive:  src-main.zip
-fb58a2082ffbeccda433271302f2dab79730101e
+Archive:  src-7906463.zip
+790646388696511b2665a4d119bf0fb713dd990d
```

Comment:
Maybe just bump to latest version because there seems to be some important fix there: https://github.com/microsoft/vscode-eslint/commit/39fdaf951e9af132255c1bd7252c14d672b60a01

Reply:
bumped the version